From: Dominik George <nik@naturalnet.de>
Subject: Use absolute path to PulseAudio modules
 The modules are installed to their locations by the xrdp-build-pulse-modules script.
--- a/instfiles/pulse/default.pa
+++ b/instfiles/pulse/default.pa
@@ -2,11 +2,11 @@
 .fail
 load-module module-augment-properties
 load-module module-always-sink
-.ifexists module-xrdp-sink.so
-load-module module-xrdp-sink
+.ifexists /var/lib/xrdp-pulseaudio-installer/module-xrdp-sink.so
+load-module /var/lib/xrdp-pulseaudio-installer/module-xrdp-sink.so
 .endif
-.ifexists module-xrdp-source.so
-load-module module-xrdp-source
+.ifexists /var/lib/xrdp-pulseaudio-installer/module-xrdp-source.so
+load-module /var/lib/xrdp-pulseaudio-installer/module-xrdp-source.so
 .endif
 load-module module-native-protocol-unix
 
--- a/sesman/chansrv/pulse/module-xrdp-sink.c
+++ b/sesman/chansrv/pulse/module-xrdp-sink.c
@@ -68,7 +68,7 @@ typedef bool pa_bool_t;
 #endif
 
 #include "module-xrdp-sink-symdef.h"
-#include "../../../common/xrdp_sockets.h"
+#include "xrdp_sockets.h"
 
 PA_MODULE_AUTHOR("Jay Sorg");
 PA_MODULE_DESCRIPTION("xrdp sink");
--- a/sesman/chansrv/pulse/module-xrdp-source.c
+++ b/sesman/chansrv/pulse/module-xrdp-source.c
@@ -55,7 +55,7 @@ typedef bool pa_bool_t;
 #endif
 
 #include "module-xrdp-source-symdef.h"
-#include "../../../common/xrdp_sockets.h"
+#include "xrdp_sockets.h"
 
 PA_MODULE_AUTHOR("Laxmikant Rashinkar");
 PA_MODULE_DESCRIPTION("xrdp source");
